journald/journald.hpp

```diff
@@ -156,6 +158,21 @@ struct Flags : public virtual flags::FlagsBase
     add(&Flags::user,
         "user",
         "The user this command should run as.");
+
+    add(&Flags::fluentbit_ip,
+        "fluentbit_ip",
+        "IP of the fluentbit listener for TCP connections.",
+        [](const Option<net::IP>& value) -> Option<Error> {
+          if (value.isNone()) {
+            return Error("--fluentbit_ip is required");
```

Comment:
Yes.  The task can choose to change the destination of its logging.  So even if the module's default is not set to `fluentbit`, we still need to have the information somewhere.

Reply:
Got it. My approach would have been to leave it optional and error out when the IP was missing while Fluent Bit was selected. But I guess, this will work just fine....